server/conf/i18n/messages

```diff
@@ -347,6 +350,8 @@ alert.eligibility_applicant_eligible_text=Based on your responses, you are likel
 alert.eligibility_applicant_not_eligible_title=You may not be eligible for this program
 # Alert text when applicant may not be eligible for a program
 alert.eligibility_applicant_not_eligible_text=Based on your responses, you may not be eligible for this program. If your information has changed, you can edit your answer and proceed with the application.
+# Alert text when applicant may not be eligible for a program (new version for North Star)
+alert.eligibilityApplicantNotEligibleText.v2=Based on your responses to the following questions, you may not be eligible for this program.
```

Comment:
@sijiayam here

Reply:
@ssandbekkhaug Are we removing `If your information has changed, you can edit your answer and proceed with the application.` for the North Star version because this alert will be displayed on the ineligible page with content that says a similar message? 

If so I would suggest we use the new string.

If that is not the case and this is an alert that displays without any other messaging about the ability to change answers and proceed, then we should keep the original string.

Reply:
> @ssandbekkhaug Are we removing `If your information has changed, you can edit your answer and proceed with the application.` for the North Star version because this alert will be displayed on the ineligible page with content that says a similar message?

Yes. The page is going to look something like this when it's done:

![ineligible](https://github.com/user-attachments/assets/b219456a-b47f-40b3-b686-52c4a40ca61c)

Here's a [link to the mocks too](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=2543-7139&m=dev).

Reply:
I should also note, the alert is used in other places, which is what I think Gwen is getting at. Here's an example:

![main-summary](https://github.com/user-attachments/assets/3203841e-c892-433c-87f8-62f39d5cbaaf)

I can imagine having `If your information has changed, you can edit your answer and proceed with the application` on the *ineligible* page would be confusing. But having that sentence on the *application review* page seems OK.

Should we have two strings?
1. A string for editable pages, such as the application review page
2. A string for non-editable pages, such as the ineligible page

Reply:
I do think we should have two strings for situations where we have supplemental information in the page content and situations where we don't.

Reply:
Ok, I updated the strings. Can we resolve this thread?

Reply:
Sounds good. Just wanted to check since we just recently changed them in the non-ns part. Thanks all!